python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -19,26 +21,21 @@
 from itertools import chain
 from functools import cached_property
 
-import warnings
-
 # numpy is always available
 import numpy as np
 
-# cuGraph or cuGraph-Service is required; each has its own version of
-# import_optional and we need to select the correct one.
+# cuGraph is required
 try:
-    from cugraph_service.client.remote_graph_utils import import_optional
+    from cugraph.utilities.utils import import_optional
 except ModuleNotFoundError:
-    try:
-        from cugraph.utilities.utils import import_optional
-    except ModuleNotFoundError:
-        raise ModuleNotFoundError(
-            "cuGraph-PyG requires cuGraph" "or cuGraph-Service to be installed."
-        )
+    raise ModuleNotFoundError("cuGraph-PyG requires cuGraph to be installed.")
+
```

Comment:
the `try`-`except` around the cugraph imports made sense before since it was requiring either cugraph or cugraph-service, but since it's only cugraph now and it's required, can we just import cugraph like all other hard dependencies?

Reply:
you're right, we can remove this.  I'll take care of it.